src/sentry/taskworker/worker.py

```diff
@@ -2,26 +2,35 @@
 
 import logging
 import time
+from concurrent import futures
 
+import grpc
 import orjson
 from django.conf import settings
 from sentry_protos.sentry.v1alpha.taskworker_pb2 import (
     TASK_ACTIVATION_STATUS_COMPLETE,
     TASK_ACTIVATION_STATUS_FAILURE,
     TASK_ACTIVATION_STATUS_RETRY,
+    DispatchRequest,
+    DispatchResponse,
 )
+from sentry_protos.sentry.v1alpha.taskworker_pb2_grpc import (
+    WorkerServiceServicer as BaseWorkerServiceServicer,
+)
+from sentry_protos.sentry.v1alpha.taskworker_pb2_grpc import add_WorkerServiceServicer_to_server
 
 from sentry.taskworker.config import TaskNamespace, taskregistry
 
 logger = logging.getLogger("sentry.taskworker")
 
 
-class Worker:
+class WorkerServicer(BaseWorkerServiceServicer):
```

Comment:
It would be good if we had a way to have the push and pull models co-exist in the repository at the same time. That would help with throughput comparisons as we could run the two options close to each other.

Perhaps we can have two Worker implementations that are toggled with CLI flags?

Reply:
Agree, added a `push` and `pull` cli command for worker and grpc